tests/cypress/certified_spec.js

```diff
@@ -0,0 +1,53 @@
+/// <reference types="cypress" />
+
+context("Certified search results", () => {
+  beforeEach(() => {
+    cy.setCookie("_cookies_accepted", "all");
+  });
+
+  it("should update results per page when pagination is selected", () => {
+    cy.visit("/certified?q=dell");
+
+    cy.findAllByLabelText("Results per page").first().select("40");
+    cy.location("href", { timeout: 6000 }).should("include", "limit=40");
+    cy.findByText(/results/).should("contain", "40");
+  });
+
+  it("should update results when Apply is selected", () => {
+    cy.visit("/certified/laptops");
+
+    cy.findByRole("checkbox", { name: /Dell/i }).click({ force: true });
+    cy.findByRole("button", { name: /Apply/i }).click();
+
+    cy.location("href", { timeout: 6000 }).should("include", "vendor=Dell");
+    cy.findAllByText(/Dell/).should("be.visible");
+  });
+
+  it("should clear filters when clear is selected", () => {
+    cy.visit("/certified/socs");
+
+    cy.findByRole("checkbox", { name: /20.04 LTS/i }).click({ force: true });
+    cy.findByRole("button", { name: /Apply/i }).click();
+    cy.findByRole("button", { name: /Clear/i }).click();
+    cy.findAllByRole("checkbox").should("not.be.checked");
+  });
+
+  it("should keep query string when filters are cleared", () => {
+    cy.visit("/certified?q=hp");
+
+    cy.findByRole("checkbox", { name: /20.04 LTS/i }).click({ force: true });
+    cy.findByRole("button", { name: /Apply/i }).click();
+    cy.findByRole("button", { name: /Clear/i }).click();
+    cy.findByPlaceholderText("Search hardware")
+      .invoke("val")
+      .should("include", "hp");
+  });
+
+  it("should take you to seach results page when search bar is used", () => {
+    cy.visit("/certified");
+
+    cy.findByPlaceholderText("Search hardware").type("dell{enter}");
+    cy.findByText(/results/).should("be.visible");
+    cy.findAllByText(/Dell/).should("be.visible");
+  });
+});
```

Comment:
I'm not sure about this, so this is more of a question. Are we running these tests somewhere in the Github actions CI?

Reply:
Not yet, I only just [fixed](https://github.com/canonical-web-and-design/ubuntu.com/pull/10191) all the current tests on ubuntu.com to ensure they all pass. I've spoken to Ant about it - he thinks it could be a good idea to run them maybe once a week on production. I'm gonna bring it up in dev catch up today

Reply:
but shouldn't this be ran on the PR everytime a dev adds new tests?